AIMSICD/src/main/res/values/translatable_strings.xml

```diff
@@ -318,7 +318,7 @@
 
 
     <!-- Section Main -->
-    <string name="help_main_neighboring_cells">Shows neighboring cells.</string>
+    <string name="help_main_neighboring_cells">Shows neighbouring cells.</string>
```

Comment:
>Why are there different English and German dialects supported by Android?

Yes, Android may do that. But what I wanted to accomplish, was a unified version of this **one** word to avoid developers using other spelling and re-introducing new inconsistencies.